tmt/package_managers/__init__.py

```diff
@@ -186,3 +186,6 @@ def install_debuginfo(
             *installables: Installable,
             options: Optional[Options] = None) -> CommandOutput:
         raise NotImplementedError
+
+    def makecache(self) -> CommandOutput:
```

Comment:
`makecache()` looks very yum-ish. `install` is easy, installs stuff, but what does `makecache()` do? We could be more verbose, e.g. `refresh_metadata` or something similar. There are many caches, often it's nothing but a buzzword, let's be more developer-friendly and announce what we do here :)